portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,34 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
+import java.util.ArrayList;
+
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/plain;");
-    response.getWriter().println("Hello Bianca!");
+    // response.setContentType("text/plain;");
+    // response.getWriter().println("Hello Bianca!");
+
+    // Convert comments to JSON
+    ArrayList<String> comments = new ArrayList<String>();
+    comments.add("This is amazing!");
+    comments.add("10/10!");
+    comments.add("LGTM");
+    String json = convertToJsonUsingGson(comments);
+
+    // Send JSON as response
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+  }
+
+  private String convertToJsonUsingGson(ArrayList<String> commentsToConvert) {
```

Comment:
nit - this method can be static:
```suggestion
  private static String convertToJsonUsingGson(ArrayList<String> commentsToConvert) {
```